src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4477,8 +4480,11 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
  /unsafe[+|-]                  Allow 'unsafe' code
  /define:&lt;symbol list&gt;         Define conditional compilation symbol(s) (Short
                                form: /d)
- /langversion:&lt;string&gt;         Specify language version mode: ISO-1, ISO-2, 3,
-                               4, 5, 6, 7.0, 7.1, 7.2, Default, or Latest
+ /langversion:?                Display the allowed values for language version
+ /langversion:&lt;string&gt;         Specify language version such as
+                               `default` (latest major version), or
+                               `latest` (latest version, including minor versions),
+                               or specific versions like `{0}` or `{1}`
```

Comment:
>`{0}` or `{1}` [](start = 57, length = 14)

Why do we need this parameterized? We can simply hardcode the versions we already support as the examples.